robot/util/rosRoverStart/rosRoverStart.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-HOME="/home/odroid"
+HOME="/home/nvidia"
```

Comment:
why is it sometimes $USER sometimes nvidia

Reply:
is this related to what you put in the description?

Reply:
There are apparently issues with using $USER when running it as a systemd service. I'll try to changing it back to see if I can get to work tho

Reply:
Also it isn't related to what I put in the description. The description refers to some weird issue with connecting to ROS over a vpn server which has a workaround.

Reply:
The systemd service doesn't work if $USER is used anywhere because the service has to run under root for some reason. I haven't find a way to fix this yet